endpoints/v1/caseStudies.json

```diff
@@ -7,7 +7,7 @@
       "vertical": "Public Sector",
       "is_enterprise": true,
       "title": "A World-First For Apple iPad",
-      "hero_image": "https://az759258.vo.msecnd.net/media/2221/decelerator_header-image-2x.jpg",
+      "hero_image": "https://raw.githubusercontent.com/theappbusiness/engineering-challenge/master/endpoints/v1/images/decelerator_header-image-2x.jpg",
```

Comment:
I _think_ this is the correct URL format, but will need to double check before merging. Some URLs contain 'blob' for GitHub, and some have `?raw=1` at the end. @samdods do you remember from when you set this up?

Reply:
this is the right format, yeah. this one has "raw.githubusercontent.com" at the domain rather than "github.com……?raw=1"